### PR TITLE
Updates new contract changes and fixes test cases

### PR DESCRIPTION
**Pull request by ShubhamChndrvnshi**
**Summary**

Updates new sumero emp contract code and corresponding test cases to work

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass

### CODE REVIEW

File: packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol [hunk 1]
```diff
@@ -907,7 +933,7 @@ contract PricelessPositionManager is Lockable {
         }
     }
 
-    function _transformPriceIdentifier(uint256 requestTime) internal view returns (bytes32) {
+    function _transformPriceIdentifier(uint256 requestTime) public view returns (bytes32) {
```
Comment (KohliSuraj) [inline packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol]: can we revert this change to match the release code?

**Review by KohliSuraj — CHANGES_REQUESTED** [review]

File: packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol (hunk 1 above)
Comment (ShubhamChndrvnshi) [reply packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol]: this is happening while committing to git. Theere is git hook configured that checks for indentation and then commits.

Comment (KohliSuraj) [reply packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol]: oh, but the hook wouldn't change internal to public? There is a way to skip the hook while committing

Comment (ShubhamChndrvnshi) [reply packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol]: yeah

Comment (KohliSuraj) [reply packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol]: @ShubhamChndrvnshi I still see this as public. Can we make this internal?

**Review by KohliSuraj — CHANGES_REQUESTED** [review]
Rest of the changes look good!

**Review by KohliSuraj — APPROVED** [review]
